gridpapers.sty

```diff
@@ -232,53 +232,46 @@
-    \pgfpathmoveto{\pgfpoint{0mm}{0.8660254*\GP@patternsize}}
-  \pgfpathlineto{\pgfpoint{\GP@patternsize}{0.8660254*\GP@patternsize}}
-    \pgfpathmoveto{\pgfpoint{0mm}{2*0.8660254*\GP@patternsize}}
-  \pgfpathlineto{\pgfpoint{\GP@patternsize}{0mm}}
-  \pgfpathlineto{\pgfpoint{0mm}{0mm}}
-    \pgfusepath{stroke}
-  }
-}
-\newcommand{\GP@declareisopat}{
-\pgfdeclarepatternformonly
-  %% Name of the pattern
-  {isometric}
-  %% Set the lower left corner of the pattern
-  {\pgfpointorigin}
-  %% Set the upper right corner of the pattern
-  {\pgfpoint{2*0.8660254*\GP@patternsize}{\GP@patternsize}}
-  %% Declare the size of the pattern blocks
-  {\pgfpoint{2*0.8660254*\GP@patternsize}{\GP@patternsize}}
-  %% Draw the pattern
-  {
-    \pgfsetlinewidth{0.6pt}
-  \pgfpathmoveto{\pgfpoint{0mm}{0mm}}
-  \pgfpathlineto{\pgfpoint{2*0.8660254*\GP@patternsize}{\GP@patternsize}}
-  \pgfpathlineto{\pgfpoint{2*0.8660254*\GP@patternsize}{0mm}}
-  \pgfpathlineto{\pgfpoint{0mm}{\GP@patternsize}}
-  \pgfpathlineto{\pgfpoint{0mm}{0mm}}
-    \pgfpathmoveto{\pgfpoint{0.8660254*\GP@patternsize}{0mm}}
-    \pgfpathlineto{\pgfpoint{0.8660254*\GP@patternsize}{\GP@patternsize}}
-    \pgfusepath{stroke}
-  }
+\tikzdeclarepattern{
+  name=triangles,
+  type=uncolored,
+  bounding box={(0,0) and (\GP@patternsize,2*0.866025*\GP@patternsize)},
+  tile size={(\GP@patternsize,2*0.866025*\GP@patternsize)},
+  parameters={\tikztrirotate},
+  tile transformation={rotate=\tikztrirotate},
+  defaults={
+    rotate/.store in=\tikztrirotate,rotate=0,
+    % tile size/.store in=\tikztilesize,tile size=10pt,
+	}, 
+  code={
+	 	% \pgfsetlinewidth{0.6pt}
+	 	% 	    \pgftransformshift{\pgfpoint{0mm}{0.866025*\GP@patternsize}}
+	 	% 	    \pgfpathmoveto{\pgfpoint{0mm}{0mm}}
+	 	% 	    \pgfpathlineto{\pgfpoint{0.5*\GP@patternsize}{0mm}}
+	 	% 	    \pgfpathlineto{\pgfpoint{\GP@patternsize}{-0.866025*\GP@patternsize}}
+	 	% 	    \pgfpathlineto{\pgfpoint{2*\GP@patternsize}{-0.866025*\GP@patternsize}}
+	 	% 	    \pgfpathlineto{\pgfpoint{2.5*\GP@patternsize}{0mm}}
+	 	% 	    \pgfpathlineto{\pgfpoint{3*\GP@patternsize}{0mm}}
+	 	% 	    \pgfpathmoveto{\pgfpoint{0.5*\GP@patternsize}{0mm}}
+	 	% 	    \pgfpathlineto{\pgfpoint{\GP@patternsize}{0.866025*\GP@patternsize}}
+	 	% 	    \pgfpathlineto{\pgfpoint{2*\GP@patternsize}{0.866025*\GP@patternsize}}
+	 	% 	    \pgfpathlineto{\pgfpoint{2.5*\GP@patternsize}{0mm}}
+	 	% 	    \pgfusepath{stroke}
+	      \pgfsetlinewidth{0.6pt}
+	    \pgfpathmoveto{\pgfpoint{0mm}{0mm}}
+	    \pgfpathlineto{\pgfpoint{\GP@patternsize}{2*0.8660254*\GP@patternsize}}
+	    \pgfpathlineto{\pgfpoint{0mm}{2*0.8660254*\GP@patternsize}}
+	      \pgfpathmoveto{\pgfpoint{0mm}{0.8660254*\GP@patternsize}}
+	    \pgfpathlineto{\pgfpoint{\GP@patternsize}{0.8660254*\GP@patternsize}}
+	      \pgfpathmoveto{\pgfpoint{0mm}{2*0.8660254*\GP@patternsize}}
+	    \pgfpathlineto{\pgfpoint{\GP@patternsize}{0mm}}
+	    \pgfpathlineto{\pgfpoint{0mm}{0mm}}
+	      \pgfusepath{stroke}
```

Comment:
Mixing tabs and spaces, let's try to avoid tabs

Reply:
Got it